src/main/java/org/schemaspy/view/HtmlMainIndexPage.java

```diff
@@ -98,7 +102,7 @@ public int compare(Table table1, Table table2) {
 
         scopes.put("tables", mustacheTables);
         scopes.put("database", database);
-        scopes.put("databaseName", databaseName);
+        scopes.put("databaseName", getDatabaseName(database));
```

Comment:
Shouldn't this line be removed and the getDatabaseName(database) be used in construction of MustasheWriter

Reply:
1. All the usages of mustachewriter.write send the databasename directly by getting the value from getDatabaseName() from HtmlFormatter
2. If the getDatabaseName() method has to be used here in MustacheWriter either the code need to be duplicated from HtmlFormatter or MustacheWriter have to extend HtmlFormatter.

Reply:
Yes that can be done. 
Let me make the changes and try.

Reply:
Hi @npetzall, 

We can pass the getDatabaseName(databse) in the MustacheWriter construction  in HtmlMainIndexPage.java but cannot remove databaseName from scope. 

This variable is used in layout of main.html and is flushed to `result` before the mainScope is added with databaseName. 

```
            Mustache mustache = mf.compile(getReader(templatePath),"template");
            mustache.execute(result, scopes).flush();
```

Reply:
Yepp, saw it now. Feels sort of awkward but it works so.

Reply:
Since if mainScope was created first it would only add unused variables to the main rendering or am I still wrong about everything :smile:, frontend is not my cup of 🍵